src/umpire/strategy/DynamicPool.cpp

```diff
@@ -33,17 +33,17 @@ DynamicPool::DynamicPool(const std::string& name,
                          const int align_bytes) noexcept :
   AllocationStrategy(name, id),
   m_allocator{allocator.getAllocationStrategy()},
-  m_min_alloc_bytes{min_alloc_bytes},
+  m_min_alloc_bytes{round_up(min_alloc_bytes, align_bytes)},
   m_align_bytes{align_bytes},
   m_coalesce_heuristic{coalesce_heuristic},
   m_used_map{},
   m_free_map{},
   m_curr_bytes{0},
-  m_actual_bytes{initial_alloc_bytes},
+  m_actual_bytes{round_up(initial_alloc_bytes, align_bytes)},
   m_highwatermark{0}
 {
-  insertFree(m_allocator->allocate(initial_alloc_bytes),
-             initial_alloc_bytes, true, initial_alloc_bytes);
+  const std::size_t bytes{round_up(initial_alloc_bytes, align_bytes)};
+  insertFree(m_allocator->allocate(bytes), bytes, true, bytes);
```

Comment:
```suggestion
  insertFree(m_allocator->allocate(rounded_to_alignment_bytes), rounded_to_alignment_bytes, true, rounded_to_alignment_bytes);
```

It does not need to be `rounded_to_alignment_bytes`, but I think that `bytes` by itself is not descriptive enough.

It could be `rounded_bytes` like it is in other parts of the code.

Reply:
Okay.